mobify/progressive-web-scaffold
# Update idk version to 0.0.4

## Changes
- Change the SDK version

## How to test-drive this PR
- Clone the scaffold and checkout this branch
- `npm install`
- `npm run configure`
- `npm run dev`
- Preview merlins potions and make sure things work
